src/data/markdown/translated-guides/en/02 Using k6/14 Scenarios/01 Executors.md

```diff
@@ -9,6 +9,19 @@ want to model to test your services.
 
 Possible values for `executor` are the executor name separated by hyphens.
 
+<Blockquote mod="note" title="VUs might not distribute uniformely over iterations">
+
+For any given scenario, you can't guarantee that a specific VU can run a specific iteration.
+
+With [`SharedArray`](/javascript-api/k6-data/sharedarray/) and [execution context variables](/using-k6/execution-context-variables/), you can map a specific VU to a specific value in your test data.
+So the tenth VU could use the tenth item in your array.
```

Comment:
```suggestion
So the tenth VU could use the tenth item in your array. Or the sixth iteration to the sixth item.
```

I would expect in most cases people care about the iteration getting *specific* or (more usually IMO) different data from the SharedArray and less about a specific VU getting one or the other.

Reply:
Yeah thanks, this is better. My main goal was to prevent readers from thinking its impossible to link VUs to specific test data, which would be a much bigger hassle